content/go/concepts/strings/terms/count/count.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Count()'
+Description: 'Returns the number of times a substring occurs in a given string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Count()`** function returns the number of times a given substring appears in a given string without overlapping.
+
+## Syntax
+
+```pseudo
+strings.Count(str, sub)
+```
+
+Where `str` is the string being searched and `sub` is the substring being searched for. It returns an `int` as the number of non-overlapping occurrences of `sub` in `str`. When `sub` is an empty string, `Count()` will count the boundaries before and after each character in `str`. (i.e. When `str` is "A" and `sub` is "", `Count()` returns 2. When `str` is "ABC" and `sub` is "", then `Count()` returns 4.)
```

Comment:
[Nitpick] Consider making the beginning part of the paragraph two separate sentences. That way, users can gain a better understanding of the roles of  `str`&  `sub` in the above code snippet. 
```suggestion
Where `str` is the string being searched.  `sub`, on the other hand, is the substring being searched for. It returns an `int` as the number of non-overlapping occurrences of `sub` in `str`. When `sub` is an empty string, `Count()` will count the boundaries before and after each character in `str`. (i.e. When `str` is "A" and `sub` is "", `Count()` returns 2. When `str` is "ABC" and `sub` is "", then `Count()` returns 4.)
```